### PR TITLE
[bash/zh-tw] add zh-tw translation

### PR DESCRIPTION
**Pull request by weihanglo**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
- [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: zh-tw/bash-tw.html.markdown [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+Contents=$(cat file.txt)
+echo "START OF FILE\n$Contents\nEND OF FILE"
+
+# 使用 `cp` 複製檔案或目錄，`cp` 會創建新版本的來源檔案／目錄
+# 所以，編輯副本不會影響到初始來源（反之亦然）。
+# 注意，如果目的地已存在該檔案／目錄，該檔案／目錄將會被覆寫
+cp srcFile.txt clone.txt
+cp -r srcDirectory/ dst/ # 遞迴複製
+
+#  `scp` or `sftp` if you plan on exchanging files between computers.
```
Comment (geoffliu) [inline zh-tw/bash-tw.html.markdown]: Delete.

**Comment by geoffliu** [comment]
👍  Just one missed line, then we can merge. Thanks!

**Comment by weihanglo** [comment]
Added missed line for `rm` command at fdc1cf9. Did you mean this?

**Comment by geoffliu** [comment]
I meant there's an extra English line you didn't delete.

**Comment by weihanglo** [comment]
Done at 7d585f8 ! Sorry for my misunderstanding.

**Comment by geoffliu** [comment]
沒關係，多謝翻譯喔

**Pull request merged**